### ISSUE TITLE
background url(./barbar.png) doesn't work

### ISSUE THREAD

**Issue by burgalon** [barbar/vortigern#61]
When using background image in CSS, e.g:
```
.home {
	text-align: center;
	background: url('./barbar.png')
}

```

the compiled CSS seems to resolve the asset fine, however the browser is trying to load it relatively to the blob:// css file which was injected to the HEAD causing an incorrect host name by Chrome

**Comment by altayaydemir** [comment]
Hello @burgalon, since we have [postcss-assets](https://github.com/assetsjs/postcss-assets) plugin, I suggest you to load images to your styles by using [its functionalities](https://css-tricks.com/images-in-postcss/).

```css
background: inline('./barbar.png');
```

**Comment by burgalon** [comment]
Hey @altayaydemir Inlining an image requires them to be base64 encoded into the CSS which isn't very efficient for big images.
postcss-assets allows also to resolve regular images and rewrite the path, however this does not seem to work well together with the current stack, possible because of the `helmet` head changes and the fact that the CSS is loaded through blob://http....

**Comment by basicsharp** [comment]
It seems that this problem occurs when we enable the `sourceMap` option on css-loader.

There are opening PRs for solving this issue in webpack/style-loader repo:

- https://github.com/webpack/style-loader/pull/124
- https://github.com/webpack/style-loader/pull/96

**Comment by OR13** [comment]
Is this issue still a thing? 

if so, can we get clear steps to reproduce?

**Comment by bherila** [comment]
Yep, I had the same issues with fonts. It was resolved by disabling the sourceMap option as @basicsharp described above.

**Comment by bherila** [comment]
So I guess this and https://github.com/barbar/vortigern/issues/142 are dupes.
